Ikuyadeu/vscode-R
# WIP Cleaning the project code for eslint (Fix  #215)

**What problem did you solve?**

Close #215

**How can I check this pull request?**

1. Get the tslint
```
sodo npm install -g eslint
```
2. Install the [eslint extensions](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
3. Edit the `.eslintrc.json`
3. You can see remained warnings
